cli/nep_test/nep17_test.go

```diff
@@ -19,19 +19,37 @@ import (
 
 func TestNEP17Balance(t *testing.T) {
 	e := testcli.NewExecutor(t, true)
+
+	args := []string{
+		"neo-go", "wallet", "nep17", "multitransfer",
+		"--rpc-endpoint", "http://" + e.RPC.Addresses()[0],
+		"--wallet", testcli.ValidatorWallet,
+		"--from", testcli.ValidatorAddr,
+		"GAS:" + testcli.TestWalletMultiAccount1 + ":1",
+		"NEO:" + testcli.TestWalletMultiAccount1 + ":10",
+		"GAS:" + testcli.TestWalletMultiAccount3 + ":3",
+	}
+	e.In.WriteString("one\r")
+	e.In.WriteString("Y\r")
+	e.Run(t, args...)
+	e.CheckNextLine(t, `^Network fee:\s*(\d|\.)+`)
+	e.CheckNextLine(t, `^System fee:\s*(\d|\.)+`)
+	e.CheckNextLine(t, `^Total fee:\s*(\d|\.)+`)
```

Comment:
Can be avoided with `--force`, but works this way too.

Reply:
Fixed.